src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -617,10 +617,14 @@ public string GetGithubRepoName(string repoUrl)
             {
                 repoName = repoName.Remove(repoName.LastIndexOf("-trusted"));
             }
-            int index = repoName.IndexOf('-');
 
             StringBuilder builder = new StringBuilder(repoName);
-            builder[index] = '/';
+
+            if (repoName.Contains("-"))
```

Comment:
indexOf returns -1 if it doesn't find it, so I don't think there's a need to do a contains + IndexOf

Reply:
fixed it. I did not pay attention that I was already getting the index.